src/main/resources/view/ResultDisplay.fxml

```diff
@@ -4,13 +4,16 @@
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.control.TextArea?>
 <?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
 
-<StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
-    <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" wrapText="true">
-      <StackPane.margin>
-         <Insets />
-      </StackPane.margin>
-      <padding>
-         <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
-      </padding></TextArea>
+<StackPane fx:id="placeHolder" minHeight="165.0" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
+   <VBox>
+      <children>
+          <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" wrapText="true">
+            <padding>
+               <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
```

Comment:
Can rethink these padding numbers? The app looks lopsided and the panel list and the view box does not start together anymore

Reply:
Yep!!